src/Console/Application.php

```diff
@@ -52,6 +53,7 @@ public function __construct()
 
         $this->toolInfo = new ToolInfo();
 
+        $this->add(new ListFilesCommand($this->toolInfo));
```

Comment:
the command is intentionally not named `ListCommand` as this would override the symfony builtin `ListCommand` and cause several unexpected side-effects

Reply:
What do you think about adding the command in alphabetical order?